bin/idl.js

```diff
@@ -333,6 +329,10 @@ function processArgs(cb) {
         return self.init(cb);
     }
 
+    if (!self.repository) {
```

Comment:
This will only occur if they run `idl` by itself? Why not invoke the help screen?